src/core/models/searchconfig.js

```diff
@@ -28,8 +34,16 @@ export default class SearchConfig {
   }
 
   validate () {
-    if (typeof this.limit !== 'number' || this.limit < 1 || this.limit > 50) {
+    if (typeof this.limitForVertical !== 'number' || this.limitForVertical < 1 || this.limitForVertical > 50) {
       throw new AnswersConfigError('Search Limit must be between 1 and 50', 'SearchConfig');
     }
+
+    if (typeof this.universalLimit === 'object' && this.universalLimit !== null) {
+      Object.values(this.universalLimit).forEach((value) => {
+        if (typeof value !== 'number' || value < 1 || value > 50) {
+          throw new AnswersConfigError('Universal limits must be between 1 and 50', 'SearchConfig');
```

Comment:
nit, I might be a little clearer to say 'Universal limits must be between 1 and 50, inclusive'. And I would add 'inclusive' to line 38 as well